pages/1.13/metrics/mesos/index.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: layout.pug
-title: Enable Mesos Metrics
-navigationTitle: Enable Mesos Metrics
+title: Mesos Metrics
+navigationTitle: Mesos Metrics
 menuWeight: 3
 excerpt: Monitoring Mesos with Telegraf
 enterprise: false
 ---
 
-You can configure DC/OS, version 1.12 or newer, to gather [observability metrics](http://mesos.apache.org/documentation/latest/monitoring/) from each Mesos agent and master. 
+The Mesos input plugin in Telegraf gathers [observability metrics](http://mesos.apache.org/documentation/latest/monitoring/) from each Mesos agent and master. The plugin is enabled by default in version 1.13 or newer.
```

Comment:
Rewrite the last line:
The Mesos input plugin is enabled by default in DC/OS 1.13 version or newer.